lib/facility_factory.rb

```diff
@@ -0,0 +1,41 @@
+class FacilityFactory
+
+    def initialize
```

Comment:
If your initialize isn't doing anything, you actually don't even need to include it! :)